app/containers/Sessions/components/QuestionSettings/Settings/Components/Tabs/BranchingTab.js

```diff
@@ -99,23 +100,33 @@ const BranchingTab = ({
   };
 
   const extraIcon = (index) => (
-    <ImageButton
-      src={copy}
-      onClick={(e) => {
-        e.stopPropagation();
-        e.preventDefault();
-        onDuplicateFormula(id, index);
-      }}
-      title={formatMessage(messages.copyFormula)}
-      disabled={disabled}
-      fill={colors.manatee}
-      iconProps={{
-        width: 16,
-        height: 16,
-        mr: 10,
-        mt: 2,
-      }}
-    />
+    <>
+      <HelpIconTooltip
+        id="formula_tooltip"
+        tooltipContent={formatMessage({
+          id: `app.GlobalMessages.formulasTooltip`,
+          defaultMessage: `For details how to construct mathematical or logical equations please visit <a href='https://www.google.com' target='_blank'>www.cias.app/resources</a>`,
+        })}
+        iconProps={{ fill: '#8C94A6' }}
```

Comment:
I wonder if we have some cias colors defined in some file? Or if not maybe we can define them and put this color there?